### PR TITLE
feat: left menu review Operations for left menu action - MEED-549 - Meeds-io/meeds#16

### PR DESCRIPTION
**Pull request by SaraBoutej**
Add Analytics to track actions in the spaces left navigation menu.

### CODE REVIEW

File: analytics-webapps/src/main/resources/locale/portlet/Analytics_en.properties [hunk 1]
```diff
@@ -324,6 +324,12 @@ analytics.editorRichOptions=Use Editor rich options
 analytics.displaySpaceHosts=Display space hosts
 analytics.backToHome=Back to home
 analytics.spaceTopbarpopover= Use space topbar popover
+analytics.spaceLeftNavigation= Use spaces left navigation menu
+analytics.left_navigation=Spaces left navigation
```
Comment (AzmiTouil) [inline analytics-webapps/src/main/resources/locale/portlet/Analytics_en.properties]:
```suggestion
analytics.spaces_left_navigation=Spaces left navigation
```

File: analytics-webapps/src/main/webapp/js/statistic-collection.js [hunk 2]
```diff
@@ -128,6 +129,21 @@ function() {
...
+      const spaceLeftNavigationAnalytics = {
+        'module': 'portal',
+        'subModule': 'ui',
+        'userId': eXo.env.portal.userIdentityId,
+        'userName': eXo.env.portal.userName,
+        'operation': eventDetail,
+        'name': 'spaceLeftNavigation',
+        'timestamp': Date.now(),
+        'parameters': {
+          'entityType': 'left_navigation',
```
Comment (AzmiTouil) [inline analytics-webapps/src/main/webapp/js/statistic-collection.js]:
```suggestion
          'entityType': 'spaces_left_navigation',
```

**Review by AzmiTouil — CHANGES_REQUESTED** [review]

File: analytics-webapps/src/main/resources/locale/portlet/Analytics_en.properties (hunk 1 above)
Comment (SaraBoutej) [reply analytics-webapps/src/main/resources/locale/portlet/Analytics_en.properties]: Done!

File: analytics-webapps/src/main/webapp/js/statistic-collection.js (hunk 2 above)
Comment (SaraBoutej) [reply analytics-webapps/src/main/webapp/js/statistic-collection.js]: Done!

**Review by AzmiTouil — APPROVED** [review]

**Pull request merged**